### PR TITLE
General MGCB Editor fixes

### PR DESCRIPTION
**Pull request by harry-cpp**

### CODE REVIEW

**Comment by SimonDarksideJ** [comment]
Any chance #7249 and #7250 could be included in this PR?

**Comment by harry-cpp** [comment]
> Any chance #7249 and #7250 could be included in this PR?

I can make a separate PR for those. I'll make a PR for whatever mgcb editor fixes I do for the day.

PS. MonoDevelop addin idea shall live again! Once I do another set of improvements...

**Comment by harry-cpp** [comment]
Since others are slow, I've included fixes for those issues in here, way too lazy to copy out the vs code config from here XD

**Comment by harry-cpp** [comment]
@tomspilman I have 9 small fixes for merge in here :)

The more you wait, the more I'll add :P

**Comment by tomspilman** [comment]
> The more you wait, the more I'll add :P

So i should wait then?  :)

File: Tools/MonoGame.Content.Builder.Editor/MainWindow.cs [hunk 1]
```diff
@@ -557,7 +544,7 @@ private void CmdDebugMode_Executed(object sender, EventArgs e)
 
         private void CmdHelp_Executed(object sender, EventArgs e)
         {
-            Process.Start("http://www.monogame.net/documentation/?page=Pipeline");
+            Process.Start(new ProcessStartInfo() { FileName = "https://docs.monogame.net/articles/tools/pipeline.html", UseShellExecute = true, Verb = "open" });
```
Comment (tomspilman) [inline Tools/MonoGame.Content.Builder.Editor/MainWindow.cs:547]: @Jjagg - I wonder how we could appropriately deal with a link like this and have it point to the right version?  I guess look at the embedded EXE version and put it into the URL?

Comment (Jjagg) [reply Tools/MonoGame.Content.Builder.Editor/MainWindow.cs:547]: I think just pointing to develop docs is best.

Comment (harry-cpp) [reply Tools/MonoGame.Content.Builder.Editor/MainWindow.cs:547]: We can easily read the version from the assembly version attribute, as long as we have docs setup correctly.

Comment (harry-cpp) [reply Tools/MonoGame.Content.Builder.Editor/MainWindow.cs:547]: Also I could really use this merged, I have a PR incoming with a questionable commit XD The URL can be fixed in a separate one.

**Comment by Jjagg** [comment]
Looking good! Merging :)

**Comment by harry-cpp** [comment]
@Jjagg Love you <3

Time for a questionable PR!

**Comment by Jjagg** [comment]
Hit me! 🤗

**Pull request merged**